Post: Invitation to Contribute (#1363)
ocaml/v3.ocaml.org: data/news/ocamlorg/invitation-to-contribute.md

---
author: Christine Rose

> title: "Contribute to the OCaml Community: Share Your Projects, Insights, and Opportunities on OCaml.org!"
> description: Invitation to encourage community members to contribute to OCaml.org

Reviewer: ```suggestion
title: "Contribute to OCaml.org: Here's How You Can Participate"
description: Share Your Projects, Insights, and Opportunities on OCaml.org!
```

### Submit a Job Opening or Book Recommendation

**Post OCaml job openings.** Are you looking to hire OCaml developers or seeking OCaml-related job opportunities? Have you come across a company looking for OCaml programmers? OCaml.org is an ideal platform to connect with talented individuals and organisations within the OCaml community. Share job openings, internships, or freelance projects, and help expand the reach of OCaml in the industry.

**Recommend OCaml books and learning resources.** Do you have a favorite OCaml book, tutorial, or learning resource that has helped you in your journey? Perhaps you have a favorite OCaml-featured blog or social network. Share it with the community! OCaml.org is an excellent place to recommend insightful books, online courses, videos, blogs, or any other materials that can empower fellow OCaml enthusiasts.

**Other ways to contribute.** Success Stories, Community Events/Meetups, and more! Instructions on how to submit the various contributions can be found in the [CONTRIBUTING.md document](https://github.com/ocaml/ocaml.org/blob/main/CONTRIBUTING.md) in the OCaml.org GitHub repo.

### Participate in OCaml's Discord Channel

> There is now a channel #ocaml.org on [the OCaml Discord server](https://discord.com/channels/436568060288172042/1126433906976112700)! If you're not yet active on Discord, please [follow this invitation link](https://discord.gg/cCYQbqN) to join the OCaml.org channel and claim your Discord profile. 

Reviewer: ```suggestion
There is now a [channel #ocaml.org on the OCaml Discord server](https://discord.com/channels/436568060288172042/1126433906976112700)! If you're not yet active on Discord, please [follow this invitation link](https://discord.gg/cCYQbqN) to join the OCaml Discord server first. 
```
